src/styles/index.scss

```diff
@@ -1,3 +1,122 @@
-body {
+.page {
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+  color: #060b35;
+
+  scroll-behavior: smooth;
+}
+
+.product-card {
+  display: flex;
+  box-sizing: border-box;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), `box-sizing: border-box;` should be set globally for all elements using the universal selector `*`, and not for individual elements. This ensures consistency across the entire project.